lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -50,11 +51,16 @@ def self.adapter_name
         end
 
         def self.tracer
+          return Datadog.tracer unless datadog_trace
           @tracer ||= datadog_trace.fetch(:tracer)
         end
 
         def self.database_service
-          @database_service ||= datadog_trace.fetch(:default_database_service, adapter_name())
+          @database_service ||= if defined?(::Sinatra)
```

Comment:
I just want to be sure that this doesn't impact the current configurations; we're already shipping major changes and I want to be sure it will not change the current Rails implementation. If this is a bugfix, good to go for me!